### PR TITLE
Format All Facts Page

### LINKED ISSUE #10: Format the Facts page
The Facts page, which lists all of the facts, needs some formatting attention. It's not terrible at the moment, but it's not done yet either.

### PR DESCRIPTION
**Pull request by UtkarshKr007**
The all facts page can be toggled between grid and list view. Right now each are in their own functions.  Next step is moving the into a component.

### CODE REVIEW

**Comment by UtkarshKr007** [comment]
The GridView and ListView are now each in a component of their own rather be function on fact page which increases code readability. Next step would be to decide the location and styling of the toggle buttons. After that individual styling of the components themselves if needed.

**Comment by floogulinc** [comment]
I wonder if its possible to remember the grid vs list view setting in memory so that when you go to a fact from grid view and then back to the list it doesn't reset the view.

**Comment by UtkarshKr007** [comment]
Forgot to co-author, but worked with @reuterShawn on making the ToggleLayout component which contains the smaller view components as well as the button controlling the view in the commit above.

**Comment by reuterShawn** [comment]
> You can revert this change.

Sorry Paul I originally undid my commit but then when I did the right changes and commited them Gitkraken asked me to pull the branch up and then it just pulled back my commit again even though I discarded the changes on my commit. Anyways I put the other margin back to 16px on another commit what a waste of build minutes.

**Review by floogulinc — APPROVED** [review]
Looks great!

File: src/pages/facts.js [hunk 1]
```diff
@@ -2,81 +2,50 @@ import React from "react"
...
+        }
+        frontmatter {
+          title
+          image {
+            src {
+              childImageSharp {
+                listImage: fixed(width: 40, height: 40) {
+                  ...GatsbyImageSharpFixed_withWebp
+                }
+                fixed(width: 190, height:150) {
```
Comment (floogulinc) [inline src/pages/facts.js]:
```suggestion
                fixed(width: 190, height: 150, quality: 70) {
```

File: src/pages/facts.js [hunk 2]
```diff
@@ -2,81 +2,50 @@ import React from "react"
...
+        excerpt(pruneLength: 50)
+        fields {
+          slug
+        }
+        frontmatter {
+          title
+          image {
+            src {
+              childImageSharp {
+                listImage: fixed(width: 40, height: 40) {
```
Comment (floogulinc) [inline src/pages/facts.js]:
```suggestion
                listImage: fixed(width: 40, height: 40, quality: 70) {
```

**Review by kklamberty — APPROVED** [review]
Looks great! Are you going to add those suggestions from @floogulinc or no? I don't know what that would do, but I do like the overall look of these views. Nice work!

**Pull request merged**